src/map/pc.c

```diff
@@ -4396,43 +4396,40 @@ int pc_payzeny(struct map_session_data *sd,int zeny, enum e_log_pick_type type,
 int pc_paycash(struct map_session_data *sd, int price, int points)
 {
 	int cash;
-	int smp;
+	int mempoints;
 	nullpo_retr(-1,sd);
 
 	points = cap_value(points,-MAX_ZENY,MAX_ZENY); //prevent command UB
-	if( price < 0 || points < 0 )
-	{
+	if( price < 0 || points < 0 ) {
 		ShowError("pc_paycash: Paying negative points (price=%d, points=%d, account_id=%d, char_id=%d).\n", price, points, sd->status.account_id, sd->status.char_id);
 		return -2;
 	}
 
-	if( points > price )
-	{
+	if( points > price ) {
```

Comment:
Thank you, seems like i always miss something. Maybe its because original code was that way.